packages/account/src/providers/provider.ts

```diff
@@ -1950,6 +1950,53 @@ Supported fuel-core version: ${supportedVersion}.`
     return bn(latestBlockHeight);
   }
 
+  /**
+   * Check if the given ID is an account.
+   *
+   * @param id - The ID to check.
+   * @returns A promise that resolves to the result of the check.
+   */
+  async isUserAccount(id: string): Promise<boolean> {
+    try {
+      const { contract, blob, transaction } = await this.operations.isUserAccount({
+        blobId: id,
+        contractId: id,
+        transactionId: id,
+      });
+
+      if (contract || blob || transaction) {
+        return false;
+      }
+      return true;
+    } catch (error) {
+      return true;
+    }
```

Comment:
@Why do we need this validation here? I wouldn't make it possible to false positives?

Reply:
Good catch 8bb3be33f